src/main/java/edu/ie3/datamodel/io/source/sql/SqlTimeSeriesMappingSource.java

```diff
@@ -9,15 +9,14 @@
 import edu.ie3.datamodel.io.factory.SimpleEntityData;
 import edu.ie3.datamodel.io.factory.timeseries.TimeSeriesMappingFactory;
 import edu.ie3.datamodel.io.naming.EntityPersistenceNamingStrategy;
-import edu.ie3.datamodel.io.naming.timeseries.IndividualTimeSeriesMetaInformation;
 import edu.ie3.datamodel.io.source.TimeSeriesMappingSource;
-import edu.ie3.datamodel.io.sql.SqlIndividualTimeSeriesMetaInformation;
 import java.util.Map;
 import java.util.Optional;
 import java.util.UUID;
 import java.util.stream.Collectors;
 
-public class SqlTimeSeriesMappingSource extends SqlDataSource<TimeSeriesMappingSource.MappingEntry>
+public class SqlTimeSeriesMappingSource
```

Comment:
The `TimeSeriesMappingSource` don't do much anymore. It might make sense to include the functionality to get the mapping in the respective sources.